command/plugin_list.go

```diff
@@ -160,9 +160,9 @@ func (c *PluginListCommand) simpleResponse(plugins *api.ListPluginsResponse, plu
 }
 
 func (c *PluginListCommand) detailedResponse(plugins *api.ListPluginsResponse) []string {
-	out := []string{"Name | Type | Version | Deprecation Status"}
+	out := []string{"Name | Type | Version | OCI Image | Runtime | Deprecation Status"}
```

Comment:
I would suggest we keep the API response as you have it but compress this output a little bit. The image and runtime are analogous to the command for a binary plugin, so condensing it down seems more consistent with the rest of the table contents to me.

```suggestion
	out := []string{"Name | Type | Version | Container | Deprecation Status"}
```